prolog/quizccc.pl

```diff
@@ -47,6 +47,9 @@
 incrementaScore(NIVEL, SCORE, NOVOSCORE) :- NIVEL =;= 2, NOVOSCORE is (SCORE + 100).
 incrementaScore(NIVEL, SCORE, NOVOSCORE) :- NIVEL =;= 3, NOVOSCORE is (SCORE + 150).
 
+decrementaLife(VIDA, NOVAVIDA) :-VIDA =\= 0, NOVAVIDA is (VIDA-1).
+decrementaLife(VIDA, NOVAVIDA) :- NOVAVIDA is -1.
```

Comment:
Quando a vida for igual a zero, devemos exibir a mensagem que o usuário perdeu o jogo, e qual o seu score atingido.